javascript/cra-starter/thirdweb.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
no need for this since we're not automating this (also we're deleting this project from this repo)